bvt/op-fvt-basic.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- IBM_PROLOG_BEGIN_TAG                                                   -->
+<!-- This is an automatically generated prolog.                             -->
+<!--                                                                        -->
+<!-- $Source: op-auto-test/bvt/op-fvt-basic.xml $                            -->
```

Comment:
I will modify the file name to op-opal-fvt.xml, op-opal-fvt-bvt.xml and also corresponding python files to op-opal-fvt.py. All opal related test functions we will be adding in these xml and python files. so that it will be easy to track and maintain these files per subsystem. If you think of better file names than above, please suggest me. so that i will use them.